jcheck/src/main/java/org/openjdk/skara/jcheck/WhitespaceCheck.java

```diff
@@ -56,7 +54,8 @@
                             var row = hunk.target().range().start() + i;
                             var tabIndex = line.indexOf('\t');
                             var crIndex = line.indexOf('\r');
-                            if (tabIndex >= 0 || crIndex >= 0 || line.endsWith(" ")) {
+                            var ignoreTab = tabPattern.matcher(path.toString()).matches();
+                            if ((tabIndex >= 0 && !ignoreTab) || crIndex >= 0 || line.endsWith(" ")) {
```

Comment:
This won't catch if a tabFile has a trailing tab.

Reply:
I'm not sure I understand. Do you want jcheck to complain about trailing tabs, even if the file type is on tabfiles/ignore-tabs?